channels/lb.m3u

```diff
@@ -3,6 +3,8 @@
 https://svs.itworkscdn.net/aghanilive/aghanilive/playlist.m3u8
 #EXTINF:-1 tvg-id="AlIttihad.lb" tvg-name="Al Ittihad" tvg-country="LB" tvg-language="Arabic" tvg-logo="http://alittihad.tv/assets/images/logo.png" group-title="General",Al Ittihad (552p)
 https://live.alittihad.tv/ittihad/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="LB" tvg-language="Arabic" tvg-logo="" group-title="Religious",Almaaref
+http://38.96.175.40:1935/liveTrans/ngrp:channel23_all/playlist.m3u8
```

Comment:
I don't seem to have any video from it.

Reply:
![image](https://user-images.githubusercontent.com/30985701/129451026-6cee4405-d046-40f4-b4dd-6338ff29cd81.png)

Reply:
logo if needed https://i.imgur.com/QYCJljN.png

but on site https://almaaref.ch/live/  i get https://5e74a9d684b2e.streamlock.net/liveTrans/ngrp:channel23_all/playlist.m3u8

https://live.almaaref.tv/  seized

Reply:
It works fine for me
https://streamtest.in/logs/http-38-96-175-40-1935-livetrans-ngrp-channel23_all-playlist-m3u8